### PR TITLE
adding annotations and resources

### PR DESCRIPTION
**Pull request by KenHuffmanAtNice**
adding annotations and resources

also fixed bugs related to a non-default .Values.agent.config.socketPath

### CODE REVIEW

**Comment by marcofranssen** [comment]
Thanks for this contribution. @KenHuffmanAtNice Could you fix the build? Changes look good.

File: charts/spire-client-example/README.md [hunk 1]
```diff
@@ -1,15 +1,18 @@
 # spire-client-example
 
+
```
Comment (marcofranssen) [inline charts/spire-client-example/README.md:3]: Remove the changes on this Chart from this PR.

**Review by marcofranssen — CHANGES_REQUESTED** [review]

File: charts/spire/Chart.yaml [hunk 2]
```diff
@@ -27,7 +27,7 @@ description: |
           - --service-account-signing-key-file=/run/config/pki/sa.key
   ```
 type: application
-version: 0.7.1
+version: 0.7.2
```
Comment (marcofranssen) [inline charts/spire/Chart.yaml:30]:
```suggestion
version: 0.7.3
```

I created another release solving a CVE with spire and bumped node-registrar.

File: charts/spire-client-example/README.md (hunk 1 above)
Comment (KenHuffmanAtNice) [reply charts/spire-client-example/README.md:3]: This change was made by helm-docs.sh script. I will revert it.

**Comment by marcofranssen** [comment]
@KenHuffmanAtNice sorry still failing. Seems you have extra new lines in the docs.

https://github.com/philips-labs/helm-charts/actions/runs/3641215496/jobs/6152018884

**Review by marcofranssen — APPROVED** [review]
LGTM 🚀 

> **Note**: Updated the docs on your behalf to get the CI succeed.

**Pull request merged**